stash_engine/app/controllers/stash_engine/admin_datasets_controller.rb

```diff
@@ -26,7 +26,12 @@ def index
     # Unobtrusive Javascript (UJS) to do AJAX by running javascript
     def status_popup
       respond_to do |format|
-        @identifier = Identifier.find(params[:id])
+        identifier = Identifier.includes(:resources).find(params[:id])
+        @curation_activity = StashEngine::CurationActivity.new(
+          identifier_id: identifier.id,
+          resource_id: identifier.latest_resource.id,
+          status: identifier.latest_resource.latest_curation_status
+        )
```

Comment:
Creates a new curation activity that can be attached to the modal form since any change made on the modal results in a new record